docs/staking-modules/csm/guides/addresses.md

```diff
@@ -0,0 +1,78 @@
+# Manager and Reward addresses
+
+When a CSM Node Operator is created, two important addresses are set: `managerAddress` and `rewardAddress`. Let's find out the purpose of these addresses, their permissions, and the management surrounding them.
+
+![addresses-1](../../../../static/img/csm/addresses-1.png)
+
+## Manager address
```

Comment:
add a note about recommended setup with hot manager address and cold reward addr

Reply:
Done